docs/application/native/getting-started/mobile/first-app.md

```diff
@@ -28,119 +22,89 @@ Studio and installing the created application on the emulator or target device.
 
-    a.  Select the **Template** project type and click **Next**.
+    1.  Select the **Template** project type and click **Next**.
 
-      ![Selecting the project        type](media/create_project_wizard_type.png)
+        ![Selecting the project type](media/create_project_wizard_type.png)
 
-    b. Select the profile (**Mobile**) and version from a drop-down
-      list and click **Next**.
+    2. Select the profile (**Mobile**) and version from a drop-down list and click **Next**.
 
-      The version depends on the platform version you have installed
-      and with which you are developing the application.
+        The version depends on the platform version you have installed and with which you are developing the application.
 
-      ![Selecting the profile and        version](media/create_project_wizard_version.png)
+        ![Selecting the profile and version](media/create_project_wizard_version.png)
 
-    c. Select the **Native Application** application type and click
-      **Next**.
+    3. Select the **Native Application** application type and click **Next**.
 
-      ![Selecting the application        type](media/create_project_wizard_app.png)
+        ![Selecting the application type](media/create_project_wizard_app.png)
 
-    d. Select the **Basic UI** template and click **Next**.
+    4. Select the **Basic UI** template and click **Next**.
 
-      ![Selecting the        template](media/create_project_wizard_template.png)
+        ![Selecting the template](media/create_project_wizard_template.png)
 
-    e. Define the project properties and click **Finish**.
+    5. Define the project properties and click **Finish**.
 
-      You can fill the project name (3-50 characters) and the unique
-      package ID. You can also select the location and working sets by  clicking **More properties**.
+        You can enter the project name (3-50 characters) and the unique package ID. You can also select the location and working sets by clicking **More properties**.
 
-      ![Defining        properties](media/create_project_wizard_properties.png)
+        ![Defining properties](media/create_project_wizard_properties.png)
 
-      The Project Wizard sets up the project, creates the application        files using the default content from the template, and closes.        For more information on the Project Wizard and the available        templates, see [Creating Tizen Projects with Tizen Project        Wizard](../../../tizen-studio/native-tools/project-wizard-n.md).
+        The Project Wizard sets up the project, creates the application files using the default content from the template, and closes. For more information on the Project Wizard and the available templates, see [Creating Tizen Projects with Tizen Project Wizard](../../../tizen-studio/native-tools/project-wizard-n.md).
```

Comment:
Don't we need to change `project-wizard-n.md` into `project-wizard.md`?

Reply:
Done